dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -347,15 +347,18 @@ ExchangeReceiverBase<RPCContext>::ExchangeReceiverBase(
 template <typename RPCContext>
 ExchangeReceiverBase<RPCContext>::~ExchangeReceiverBase()
 {
+    bool wait_connection_done = false;
     try
     {
         close();
         waitAllConnectionDone();
```

Comment:
just `ExchangeReceiverBase<RPCContext>::close() noexcept` and `ExchangeReceiverBase<RPCContext>::waitAllConnectionDone() noexcept` is ok.

Reply:
> just `ExchangeReceiverBase<RPCContext>::close() noexcept` and `ExchangeReceiverBase<RPCContext>::waitAllConnectionDone() noexcept` is ok.

okk